composer.json

```diff
@@ -19,7 +19,7 @@
     "require": {
         "php": "^7.2",
         "doctrine/annotations": "~1.0",
-        "jms/serializer": "^2.1|^3.0",
+        "jms/serializer": "^2.1 || ^3.2",
```

Comment:
can you explain the reason for this change?

Reply:
jms/serializer 3.0 and 3.1 cause some php7.4 deprecation warnings. But I realized that my change may not really makes sense as jms/serializer 2.1 will cause them, too. 

Do you have an idea how this could be solved for the test using jms/serializer 2.1? Maybe we should make jms/serializer 2.1 php7.4 ready, too?

Reply:
jms/serializer 2.x is EOL, so i thing we can drop it.

Reply:
I've dropped jms/serializer 2.x in ea7e5f7238d34a365311f467f8bf7e642ec1b366